app/views/homepage/index.html.erb

```diff
@@ -85,27 +85,27 @@
             <div class="promo-image">
               <div class="promo-content">
                 <a href="/guidance/grenfell-tower-fire-june-2017-support-for-people-affected">
-                  <%= image_tag 'homepage/grenfell-tower-support.png', alt: '' %>
+                  <%= image_tag 'homepage/grenfell-tower-support.png', alt: 'grenfell tower support' %>
```

Comment:
The alt text is intentionally empty because the image is decorative. The description doesn't add valuable information for screen readers and duplicates the title. See https://github.com/alphagov/frontend/issues/1059 and https://github.com/alphagov/frontend/pull/1313 for more.

Reply:
thanks @tijmenb - undid the alts in a0c0447